packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -32,40 +36,62 @@ export interface IOutboxParameters {
 }
 
 export class Outbox {
+	private readonly mc: MonitoringContext;
 	private readonly attachFlowBatch: BatchManager;
 	private readonly mainBatch: BatchManager;
-	private readonly defaultAttachFlowSoftLimitInBytes = 64 * 1024;
+	private readonly defaultAttachFlowSoftLimitInBytes = 320 * 1024;
```

Comment:
This number is the limit before compression, right? It's very hard to say what algorithm we should follow to find "best" number here. It feels like if app is using timer-based flush, they tell us - they do not care about latency, but care much more about batch counts. And if so, we might use this hint to substantially increase this limit, or maybe even - eliminate attach ops sending ahead of time (as grouping all ops into smaller set of batches will result in better througtput / lower cost for consumers, at the expense of higher latency.

Reply:
> This number is the limit before compression, right? 

Yes. I think we discussed this being too aggressive.